packages/fxa-settings/src/models/Account.ts

```diff
@@ -273,6 +274,18 @@ export class Account implements AccountData {
     return this.data.passwordCreated;
   }
 
+  get hasPassword() {
+    // This might be requested before account data is ready,
+    // so default to disabled until we can get a proper read
+    try {
+      return (
+        this.data?.passwordCreated != null && this.data.passwordCreated > 0
```

Comment:
Should this be `!== null`? But, nice👌, let's get your PR in so I can rebase and use this instead of my `hasPassword = passwordCreated > 0` check in `Security/index.tsx` in my PR.

Reply:
I think it should be != which covers both the null case and the undefined case.